dashboard/config/initializers/devise.rb

```diff
@@ -315,6 +315,20 @@
     manager.failure_app = CustomDeviseFailure
   end
 
+  Warden::Manager.after_authentication do |user, auth|
+    if auth.cookies[:pm] == "new_header"
+      cookie_key = '_user_type' + (Rails.env.production? ? '' : "_#{Rails.env}")
```

Comment:
I actually just did this to match the existing `_learn_session` cookie, but I'm not sure how it got that naming convention.